### PR TITLE
Tests for lq ws

### PR DESCRIPTION
**Pull request by phughk**
## What is the motivation?

Test lq against live endpoint and query endpoint and validate notifications received

## What does this change do?

Add ws tests for live queries using live endpoint and query endpoint

## What is your testing strategy?

`cargo test --package surreal --bin surreal --no-default-features --features storage-mem,http --test ws_integration live -- --nocapture`

## Is this related to any issues?

LQ

## Have you read the [Contributing Guidelines](https://github.com/surrealdb/surrealdb/blob/main/CONTRIBUTING.md)?

- [x] I have read the [Contributing Guidelines](https://github.com/surrealdb/surrealdb/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: tests/common/mod.rs [hunk 1]
```diff
@@ -190,6 +195,20 @@ pub async fn ws_send_msg(
...
+			_ = time::sleep(time::Duration::from_millis(2000)) => {
+					return Err(TestError::NetworkError{message: "timeout waiting for the response".to_string()}.into());
+			}
+			msg = f.select_next_some() => {
+					serde_json::from_str(&msg?.to_string())?
+			}
+	};
+	Ok(serde_json::from_str(&msg.to_string())?)
+}
+
```
Comment (sgirones) [inline tests/common/mod.rs:211]: could we reuse the code here for `ws_send_msg_with_fmt`? Maybe have a `ws_recv_msg_with_fmt()` that we use here and in the other method

File: tests/ws_integration.rs [hunk 2]
```diff
@@ -323,10 +325,174 @@ async fn kill() -> Result<(), Box<dyn std::error::Error>> {
 	Ok(())
 }
 
-#[test(tokio::test)]
+#[tokio::test]
```
Comment (sgirones) [inline tests/ws_integration.rs:328]: I guess this change was not intentional. Can you restore to the previous line?

File: tests/ws_integration.rs [hunk 3]
```diff
@@ -323,10 +325,174 @@ async fn kill() -> Result<(), Box<dyn std::error::Error>> {
...
+		result["name"].as_str().unwrap(),
+		serde_json::to_value("ok").unwrap(),
+		"result: {:?}",
+		res
+	);
+
+	Ok(())
+}
+
+#[tokio::test]
```
Comment (sgirones) [inline tests/ws_integration.rs:415]:
```suggestion
#[test(tokio::test)]
```

File: tests/ws_integration.rs (hunk 2 above)
Comment (sgirones) [reply tests/ws_integration.rs:328]:
```suggestion
#[test(tokio::test)]
```

**Comment by phughk** [comment]
Closing as was merged with main

**Pull request closed without merging**